quickstart.md

```diff
@@ -132,23 +132,29 @@ kubectl get runtimeclass
 ```
 Output:
 ```
-NAME            HANDLER         AGE
-kata            kata            9m55s
-kata-clh        kata-clh        9m55s
-kata-clh-tdx    kata-clh-tdx    9m55s
-kata-qemu       kata-qemu       9m55s
-kata-qemu-tdx   kata-qemu-tdx   9m55s
-kata-qemu-sev   kata-qemu-sev   9m55s
+NAME                 HANDLER              AGE
```

Comment:
@mythi should there be enclave-cc here?

Reply:
Good point. I think it would make sense since `enclave-cc.md` does not cover it and it can also co-exist with `kata-qemu-tdx`.